src/bin/inx-chronicle/api/mod.rs

```diff
@@ -59,24 +53,15 @@ impl ApiWorker {
         Ok(Self {
             db: db.clone(),
             api_data: config.clone().try_into()?,
-            server_handle: None,
         })
     }
-}
-
-#[async_trait]
-impl Actor for ApiWorker {
-    type State = ();
-
-    type Error = ApiError;
 
-    async fn init(&mut self, cx: &mut ActorContext<Self>) -> Result<Self::State, Self::Error> {
-        let (sender, receiver) = oneshot::channel();
+    // TODO: What happens when there is an error in an API request. This should not bring down all of Chronicle.
```

Comment:
I don't think errors in requests will ever stop the axum server.

Reply:
Ok, great!